sdk/extensions/wcf/Microsoft.WCF.Azure.StorageQueues/README.MD

```diff
@@ -0,0 +1,94 @@
+# WCF Azure Queue Storage client library
+
+WCF Azure Queue Storage client is the client side library that will help WCF clients to be able to use Azure Queue Storage instead of using MSMQ.
+
+## Getting started
+
+### Install the package
+
+Install the Azure.WCF.AzureQueueStorage.Client library for .NET with [NuGet][nuget]:
+
+```dotnetcli
+dotnet add package Azure.WCF.AzureQueueStorage.Client
+```
+
+### Prerequisites
```

Comment:
There are a bunch of sections talking about AppConfig here as well.   Is that intentional?

Reply:
This should all be fixed now